### PR TITLE
Fuzz pvc.Spec.Selector in API tests

### PR DESCRIPTION
**Pull request by pmorie**
Fuzz PersistentVolumeClaim.Spec.Selector

Follow-up to #25917

### CODE REVIEW

File: pkg/api/testing/fuzzer.go [hunk 1]
```diff
@@ -357,6 +357,11 @@ func FuzzerFor(t *testing.T, version unversioned.GroupVersion, src rand.Source)
 		},
 		func(pvc *api.PersistentVolumeClaim, c fuzz.Continue) {
 			c.FuzzNoCustom(pvc) // fuzz self without calling this function again
+			pvc.Spec.Selector = &unversioned.LabelSelector{
```
Comment (lavalamp) [inline pkg/api/testing/fuzzer.go]: Would it make sense to write a custom fuzzer for unversioned.LabelSelector?

I can't tell if this is adding coverage or removing coverage.

Comment (pmorie) [inline pkg/api/testing/fuzzer.go]: @lavalamp there's already a TODO for it elsewhere in the file -- I would do it in a follow up if you wanted but it seemed like quite a bit of work -- this at least tests that the selelector goes through the conversion appropraitely.

Comment (lavalamp) [inline pkg/api/testing/fuzzer.go]: It should be as easy as adding a function like this? Then you'll test it everywhere it appears.

```go
...,
func(uls *unversioned.LabelSelector, c fuzz.Continue) {
  uls.MatchLabels = map[string]string{
    "testlabelkey": "testlabelval",
  }
},
...
```

Are you sure it's not tested now? I'm pretty sure that the fuzzer recurses into map types.

**Comment by dchen1107** [comment]
ping ...

**Comment by pmorie** [comment]
I don't think this PR is needed, actually.  After looking in the fuzzer, I think this removes instead of adds coverage.  Closing.  In a follow up, we should remove the logic for another test that does similar manual fuzzing for `unversioned.LabelSelector`.  Sorry for the churn.

**Pull request closed without merging**